app/upgrades/v2/upgrades.go

```diff
@@ -20,7 +21,7 @@ var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
 	StoreUpgrades: store.StoreUpgrades{
-		Added:   []string{tokenfactorytypes.ModuleName, pfmroutertypes.StoreKey},
+		Added:   []string{tokenfactorytypes.ModuleName, pfmroutertypes.StoreKey, ratelimittypes.StoreKey},
```

Comment:
I am wondering, when adding ratelimiter through upgrade what are default settings for our already existing routes  ? Do wee need to configure them in upgrade or do we need to pass some upgrade proposal just after upgrade ?

Reply:
Makes sense! 

I guest to add the default parameters we need to setup them in upgrade similiarty to https://github.com/babylonlabs-io/babylon/pull/897/files#diff-a4f05ee252c14e4fea8505ea6cbd3791d4e43e38ecf7daac33367a90f5c3966a ?